src/App.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
+import Homepage from './components/Homepage';
 
 function App() {
   return (
     <div className="App">
+      <Homepage />
       <Routes>
-        <Route path="/" element={<div>Hello</div>} />
+        <Route path="/" />
```

Comment:
- [ ] [REQUIRED] Please, add the Homepage page in the home route path because it will be difficult to do the upcoming tasks with this element located outside a route. It should look like this `<Route path="/" element={<Homepage />} />`